fjord/base/tests/test_google_utils.py

```diff
@@ -77,24 +80,32 @@ def test_ga_track_event_non_test(self):
                 }
             )
 
+    @set_sys_argv([])
     def test_ga_track_event_async_true(self):
         """async=True, then delay() gets called once"""
         with requests_mock.Mocker() as m:
             m.post(GOOGLE_API_URL, text='')
 
-            d = 'fjord.base.google_utils.post_event.delay'
-            with patch(d) as delay_patch:
-                params = {'cid': 'xxx', 'ec': 'ou812'}
-                ga_track_event(params, async=True)
-                assert delay_patch.call_count == 1
+            post_event = 'fjord.base.google_utils.post_event'
+            post_event_delay = 'fjord.base.google_utils.post_event.delay'
+            with patch(post_event) as post_event_patch:
+                with patch(post_event_delay) as post_event_delay_patch:
+                    params = {'cid': 'xxx', 'ec': 'ou812'}
```

Comment:
Not a problem, but in python 2.7 and above (I think), you could do

```python
with patch(post_event) as post_event_patch, patch(post_event_delay) as post_event_delay_patch:
    params = ...
```

That might blow your line length limit though, which would sort of ruin the point. *shrug*